### DISCUSSION TITLE
How can we import users with encrypted passwords?

### DISCUSSION THREAD

**Discussion by aovasylenko** [zitadel/zitadel#4001] [Q&A] [answered]
Do we have a list of supported password hash methods, so that we can verify, that we can import users with the need to reenter their password? 
Thanks in advance!

**Comment by hifabienne** [comment, marked as answer]
Hi @aovasylenko 

We don't have a docs page for this right now.
All our passwords are encrypted with bcrypt right now.
At the moment we do not provide an import of encrypted passwords.
With the migration from ZITADEL v1 to v2 we have implemented an import endpoint to import all ressources, this import endpoint allows you to import bcrypted passwords.
